src/webviews/src/index.tsx

```diff
@@ -28,9 +36,30 @@ window.addEventListener('message', e => {
 //   data: {foo: true}
 // })
 
+const rootEl = document.getElementById('root')
+
+function AppRoutes() {
+  let location = useLocation()
+  let navigate = useNavigate()
+  useEffect(() => {
+    navigate(`/${rootEl.dataset.name}`, { replace: true })
```

Comment:
If I'm understanding the goal here, we could probably use the `MemoryRouter` instead of `BrowserRouter` and "seed" it with the initial route that we want 👇 

```tsx
<MemoryRouter initialEntries={["something"]} initialIndex={0} />
```

Reply:
Nice, addressed